polyswarm_challenge/solution.py

```diff
@@ -0,0 +1,117 @@
+import binascii
+from manticore.ethereum import ManticoreEVM, ABI
+
+m = ManticoreEVM()
+
+owner_account      = m.create_account(balance=1000, name='owner',     address=0xbc7ddd20d5bceb395290fd7ce3a9da8d8b485559)
+attacker_account   = m.create_account(balance=1000, name='attacker',  address=0x762C808237A69d786A85E8784Db8c143EB70B2fB)
+cashmoney_contract = m.create_account(balance=1000, name='CashMoney', address=0x64ba926175bc69ba757ef53a6d5ef616889c9999)
+
+bytecode = binascii.unhexlify(b"7f646f6765636f696e746f7468656d6f6f6e6c616d626f736f6f6e64756465732160015560c06040526020"
```

Comment:
Can we move the bytecode to a file and just read its content?

Reply:
Fixed